From 83c345908ec1006f0734ad674578eecbc075e691 Mon Sep 17 00:00:00 2001
From: Paul Mundt <lethal@linux-sh.org>
Date: Wed, 20 Jun 2012 22:29:14 +0900
Subject: sh: pfc: Kill off unused pinmux bias flags.

WANT_PULLUP/DOWN were never interfaced with anything, so just kill them
off.

Signed-off-by: Paul Mundt <lethal@linux-sh.org>
(cherry picked from commit 9f65b2b60c42c1e2d885acede4443b53f141c987)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 include/linux/sh_pfc.h |    2 --
 1 file changed, 2 deletions(-)

diff --git a/include/linux/sh_pfc.h b/include/linux/sh_pfc.h
index 95dad34..c06a473 100644
--- a/include/linux/sh_pfc.h
+++ b/include/linux/sh_pfc.h
@@ -25,8 +25,6 @@ typedef unsigned short pinmux_flag_t;
 #define PINMUX_TYPE_INPUT_PULLDOWN  6
 
 #define PINMUX_FLAG_TYPE            (0x7)
-#define PINMUX_FLAG_WANT_PULLUP     (1 << 3)
-#define PINMUX_FLAG_WANT_PULLDOWN   (1 << 4)
 
 #define PINMUX_FLAG_DBIT_SHIFT      5
 #define PINMUX_FLAG_DBIT            (0x1f << PINMUX_FLAG_DBIT_SHIFT)
-- 
1.7.10.4

